docs/tutorials/quick-start/index.rst

```diff
@@ -702,25 +756,106 @@ certificate.
         Name:       letsencrypt-prod
       Secret Name:  quickstart-example-tls
     Status:
-      Acme:
-        Order:
-          URL:  https://acme-v02.api.letsencrypt.org/acme/order/45980184/182533829
       Conditions:
-        Last Transition Time:  2018-11-19T19:16:10Z
-        Message:               Certificate issued successfully
-        Reason:                CertIssued
-        Status:                True
-        Type:                  Ready
-        Last Transition Time:  <nil>
-        Message:               Order validated
-        Reason:                OrderValidated
+        Last Transition Time:  2019-01-09T13:52:05Z
+        Message:               Certificate does not exist
+        Reason:                NotFound
         Status:                False
-        Type:                  ValidateFailed
+        Type:                  Ready
+    Events:
+      Type    Reason        Age   From          Message
+      ----    ------        ----  ----          -------
+      Normal  Generated     18s   cert-manager  Generated new private key
+      Normal  OrderCreated  18s   cert-manager  Created Order resource "quickstart-example-tls-889745041"
+
+You can monitor the progress of the ACME Order by running ``kubectl describe``
+on the Order resource that cert-manager has created for your Certificate:
+
+.. code-block:: shell
+
+    $ kubectl describe order quickstart-example-tls-889745041
+    ...
+    Events:
+      Type    Reason      Age   From          Message
+      ----    ------      ----  ----          -------
+      Normal  Created     90s   cert-manager  Created Challenge resource "quickstart-example-tls-889745041-0" for domain "example.your-domain.com"
+
+Here, we can see that cert-manager has created 1 'Challenge' resource to fulfil
+the Order. You can dig into the state of the current ACME challenge by running
+``kubectl describe`` on the automatically created Challenge resource:
+
+.. code-block:: shell
+
+    $ kubectl describe challenge quickstart-example-tls-889745041-0
+    ...
+
+    Status:
+      Presented:   true
+      Processing:  true
+      Reason:      Waiting for http-01 challenge propagation
+      State:       pending
+    Events:
+      Type    Reason     Age   From          Message
+      ----    ------     ----  ----          -------
+      Normal  Started    15s   cert-manager  Challenge scheduled for processing
+      Normal  Presented  14s   cert-manager  Presented challenge using http-01 challenge mechanism
+
+From above, we can see that the challenge has been 'presented' and cert-manager
+is waiting for the challenge record to propagate to the ingress controller.
+You should keep an eye out for new events on the challenge resource, as a
+'success' event should be printed after a minute or so (depending on how fast
+your ingress controller is at updating rules):
```

Comment:
This would be an ideal place to reference what the states of a challenge are and what you might expect, if that's captured elsewhere in any other documentation - might be a good place to link to if that exists...

Reply:
I've opened #1218 which adds these docs - I'll update this PR with a link to it once it has merged.

Reply:
Done, I've linked to the reference docs for now. We are putting together a 'troubleshooting' guide soon too, which will probably be another good resource to link users to as well.